worker/src/test/scala/org/apache/celeborn/service/deploy/MiniClusterFeature.scala

```diff
@@ -52,7 +52,7 @@ trait MiniClusterFeature extends Logging {
     val conf = new CelebornConf()
     conf.set(CelebornConf.METRICS_ENABLED.key, "false")
     val prometheusPort = masterPrometheusPort.getAndIncrement()
-    conf.set(CelebornConf.MASTER_PROMETHEUS_PORT.key, s"$prometheusPort")
+    conf.set(CelebornConf.MASTER_HTTP_SERVICE_PORT.key, s"$prometheusPort")
     logInfo(s"set celeborn.master.metrics.prometheus.port to $prometheusPort")
     if (map != null) {
```

Comment:
the log message should be changed，at the same time, the variable names masterPrometheusPort and workerPrometheusPort should remain consistent with the configuration items.